docs/contributing/index.rst

```diff
@@ -3,59 +3,26 @@
 Contributing to Weblate
 =======================
 
+Language
+--------
 
-There are dozens of ways to contribute in Weblate. Any help is welcomed, be it
-coding, graphics design, documentation or sponsorship:
+`Translate <https://hosted.weblate.org/projects/weblate/>`_ Weblate itself,
+and point out errors in source strings while you do. Suggest source string changes, or
+click "Source string location" in the lower right of the regular editor to have
+a go yourself.
 
-* :doc:`issues`
-* :doc:`start`
-* :ref:`translating-weblate`
-* :ref:`contributing-docs`
-* :ref:`funding-weblate`
+Here in the documentation, click :guilabel:`Edit on GitHub` atop the page to do the same thing.
+`Open an issue https://github.com/WeblateOrg/weblate/issues/new?assignees=comradekingu&labels=ux&template=bug_report.md>`_
```

Comment:
Are documentation issues tagged anything other than "ux"?

Reply:
Yes, there’s a label `documentation`. Thanks for the head‘s up, we forgot to use it before.

There is no need to hard-code assigning link into the docs, but we have a team [`reviewers`](https://github.com/orgs/WeblateOrg/teams/reviewers) you are a member of.